integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt

```diff
@@ -248,8 +255,11 @@ class Sep24End2EndTest(config: TestConfig, val jwt: String) {
     assertNotNull(actualEvents)
     actualEvents?.let {
       assertEquals(5, it.size)
-      val expectedEvents: List<AnchorEvent> =
-        gson.fromJson(expectedWithdrawEventJson, object : TypeToken<List<AnchorEvent>>() {}.type)
+      val expectedEvents: List<SendEventRequest> =
+        gson.fromJson(
+          expectedWithdrawEventJson,
+          object : TypeToken<List<SendEventRequest>>() {}.type
+        )
       compareAndAssertEvents(asset, expectedEvents, actualEvents)
```

Comment:
TBH I'm not sure how this even works today, same for the same code in the deposit test.

`expectedEvents` are hard-coded in these tests, including IDs, hashes, timestamps, etc. How are the real events created throughout the transaction flow equal to the hard-coded transactions in this file?

Reply:
The `expectedWithdrawEventJson` has the fields that does not change. The dynamic fields will be added in the `compareAndAssertEvents` function.

The `gson.fromJson` part is the way to read JSON to a list of objects.